src/audio/load.c

```diff
@@ -64,7 +64,7 @@ Acmd* alAdpcmPull(void* filter, s16* outp, s32 outCount, s32 sampleOffset, Acmd*
     inp = AL_DECODER_IN;
     aLoadADPCM(ptr++, f->bookSize, K0_TO_PHYS(f->table->waveInfo.adpcmWave.book->book));
 
-    looped = (outCount + f->sample > f->loop.end) && (f->loop.count != 0);
+    looped = ((int)(outCount + f->sample) > (int)(f->loop.end)) && (f->loop.count != 0);
```

Comment:
What was the verdict on these comparisons? Are they equivalent to the originals?